deployments/warp_routes/ETH/arbitrum-conwai-config.yaml

```
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=../schema.json
+tokens:
+  - addressOrDenom: "0x1A59A0d9CA5b500942F46D3bF75B6E4410531849"
+    chainName: arbitrum
+    coinGeckoId: arbitrum
+    connections:
+      - token: arbitrum|conwai|0x1A59A0d9CA5b500942F46D3bF75B6E4410531849
```

how did you craft these configs? the first value is "protocol" indicating the type of VM, the second value is the chain. don't think our CLI would've exported it like this
```suggestion
      - token: ethereum|conwai|0x1A59A0d9CA5b500942F46D3bF75B6E4410531849
```